scout/server/blueprints/clinvar/controllers.py

```diff
@@ -325,15 +325,21 @@ def update_clinvar_submission_status(request_obj, institute_id, submission_id):
             f"Removed {deleted_objects} objects and {deleted_submissions} submission from database",
             "info",
         )
+    if update_status == "submit":
+        submitter_key = request_obj.form.get("apiKey")
+        send_api_submission(institute_id, submission_id, submitter_key)
 
 
-def validate_submission(submission_id):
-    """Validate a submission object and documents using the ClinVar API
+def json_api_submission(submission_id):
+    """Converts submission objects (Variant and Casedata database documents) to a json submission using
```

Comment:
Moved the code that converts 2 csv files in the json submission in a separate function, so it can be invoked by validate_submission and send_api_submission (this step is in both)